use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -296,33 +319,147 @@ main(int argc, char *argv[])
     printf("Start Server, using clock '%s'"
            ": Initiate L3-%slogging to log-file '%s'"
            ", using %s encoding scheme.\n",
-           clock_name(clock_id), l3_log_mode, logfile, loc_scheme);
+           clock_name(Clock_id), l3_log_mode, logfile, loc_scheme);
 
     snprintf(run_descr, sizeof(run_descr), "L3-%slogging %s",
              l3_log_mode, loc_scheme);
 
 #else // L3_ENABLED
 
-    printf("Start Server, using clock ID=%d '%s': No logging.\n",
-           clock_id, clock_name(clock_id));
+    printf("Start Server, using clock ID=%d '%s': No logging"
+           ", %d server-threads, server-msgq-ID=%d.\n",
+           Clock_id, clock_name(Clock_id), num_threads, serverId);
     snprintf(run_descr, sizeof(run_descr), "Baseline - No logging");
 
 #endif // L3_ENABLED
 
-    /* Read requests, handle each in a separate child process */
-    Client_info *clientp = NULL;
-    requestMsg req;
-    responseMsg resp;
+    // Invoke pthread_create() to create n-threads ...
+    svr_thread_config   svr_config = {
+                            .server_id = serverId
+                        };
+    pthread_t   thread_ids[num_threads];
 
     struct timespec ts0 = {0};
-    struct timespec ts1 = {0};
 
+    if (svr_start_threads(thread_ids, num_threads, (void *) &svr_config)) {
+        errExit("Server thread creation failed.");
+    }
 #if !defined(__APPLE__)
-    if (clock_gettime(clock_id, &ts0)) {    // ***** Timing begins
+    if (clock_gettime(Clock_id, &ts0)) {        // ***** Timing begins
```

Comment:
**NOTE**: The clock-begin is happening -after- the server thread has been started.

In future, when this is enhanced to actually support n-threads, we will start-up the threads, each of which will sit around waiting in `msgrecv()`. The timer-begin will happen sometime after that.

So, there is a bit of fuzziness in exactly when the clock will begin, but I think it should average out over millions of requests processed.

Let me know if you think this timing-begin logic needs to be tightened more.